test/res.render.js

```diff
@@ -110,7 +111,10 @@ describe('res', function(){
 
         request(app)
         .get('/')
-        .expect(/Cannot read property '[^']+' of undefined/, done);
+        .expect(jsEngine.engineSpecificMessage({
```

Comment:
I would just replace the `res.end(err.message)` above with `res.send('caught error: ' + err.message')` and then the expect could just be `.expect(/^caught error: /, done)` since that would tell us we got the error in next(err)